boards/tenstorrent/tt_blackhole/init.c

```diff
@@ -83,6 +83,10 @@ static int flash_reset_init(void)
 
 	if ((spi_device_config.f.normal_spi_mode == SpiOctalMode) &&
 	    (spi_device_id == 0x2c5b1a10)) {
+		if (spi_device_config.f.normal_ddr) {
```

Comment:
Testing on the P100 in CI, it *seems* like this might have been the root cause of P100 failures in CI recently? Not 100% sure, but the flash write test failed locally without this change, but passed with it added

Reply:
P100 flash test passed locally with this change, but still fails in CI